src/main/resources/templates/lfp/home.html

```diff
@@ -65,11 +65,15 @@ <h2 class="govuk-heading-s">If you're paying from an overseas account</h2>
                 <p class="govuk-body">You can pay through a bank transfer from an overseas account.</p>
                 <p class="govuk-body">You'll need to use Companies House swift code, account number and account name.</p>
```

Comment:
Maybe a daft question but does this wording need a slight tweak as it says use swift code, account number and account name but we've added sort code and IBAN?